Sources/OpenTelemetryApi/Metrics/Stable/DefaultStableMeter.swift

```diff
@@ -123,18 +123,18 @@ public class DefaultStableMeter : StableMeter {
     func add(value: Int, attribute: [String : AttributeValue]) {}
   }
   
-  private class NoopLongCounterBuilder : LongCounterBuilder {
-    func ofDoubles() -> DoubleCounterBuilder {
-      NoopDoubleCounterBuilder()
-    }
-    
-    func build() -> LongCounter {
-      NoopLongCounter()
-    }
-    
-    func buildWithCallback(_ callback: @escaping (ObservableLongMeasurement) -> Void) -> ObservableLongCounter {
-      NoopObservableLongCounter()
-    }
+    private class NoopLongCounterBuilder : LongCounterBuilder {
+        func ofDoubles() -> DoubleCounterBuilder {
```

Comment:
I cannot find the utility of this method in the PR

Reply:
There is no code change, it just indent change here